File: C/Assignment 5/inverted RAT.c
// inverted RAT
#include<stdio.h>
void main()
{
	int no=5;
	for(int row=no;row>=1;row--)
	{
		for(int col=1;col<=row;col++)
		{
			printf("*");
		}
		printf(" \n");
	}
}

File: OOP/Assignment 5/Inverted_rat.java
class Inverted_rat
{
public static void main(String args[])
{
	int no=5;
	for(int row=no;row>=1;row--)
	{
		for(int col=1;col<=row;col++)
		{
			System.out.printf("*");
		}
		System.out.printf(" \n");
	}
}
}